<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphwindow</class>
 <widget class="QDialog" name="Graphwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(145, 52, 162, 92);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1" rowminimumheight="1,1,1,0">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>Функция :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="function_is">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(250, 250, 250);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgba(37, 37, 37, 117);
    color:  rgb(255, 255, 255);
    font-size:  33px;
	font: 16pt &quot;Comic Sans MS&quot;;
}

QPushButton:pressed {
    background-color:  rgb(148, 59, 165);
}</string>
         </property>
         <property name="text">
          <string>Нарисовать график</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="4">
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string> x =</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>62</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>X max =</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="x_min">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>-1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="x_max">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QDoubleSpinBox" name="x_is">
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="minimum">
          <double>-99999.990000000005239</double>
         </property>
         <property name="maximum">
          <double>99999.990000000005239</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>62</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>X min =</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="5">
        <widget class="QLabel" name="fx_is">
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="find_fx">
         <property name="maximumSize">
          <size>
           <width>53</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgba(37, 37, 37, 117);
    color:  rgb(255, 255, 255);
    font-size:  33px;
	font: 14pt &quot;Comic Sans MS&quot;;
}

QPushButton:pressed {
    background-color:  rgb(148, 59, 165);
}</string>
         </property>
         <property name="text">
          <string>f(x) =</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="y_min">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>-1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>62</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>Y min =</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="y_max">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>62</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>Y max =</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QCustomPlot" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(144, 50, 161, 48);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
